src/model/constants.py

```diff
@@ -0,0 +1,9 @@
+headers = {'Authorization': 'token '}
+
+token = input('Github token: ')
```

Comment:
provide a name of environment variable to read github token from - it's more secure way to do it. Anyway - it should be part of CLI instead of asking it during import
If you are creating constants - let's follow pep8 naming convention for constants - it should be UPPER_CASE

Reply:
and (it looks like) all these constants could be default values to CLI

Reply:
about setting token in environment variable - i'm not sure it's the best solution, and my colleagues seem to agree. It's supposed to be generated for one-time use, that's what should make it secure